mobile/lib/screens/on_boarding/splash_screen.dart

```diff
@@ -14,6 +14,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
+import '../../main_common.dart';
```

Comment:
Hi @katendejericho5 ,do we have any Flutter tool which enables one to be able to avoid using these relative paths? I am sensing that things can get bloody? I have done some quick research and learnt that:

1.  you can actually utilise the package key word like import 'package:my_flutter_project/src/my_module/my_module.dart';
2. that the lib directory is the default root of your Dart package, and you don't need to specify it in the import statement. Just start with the path relative to the lib directory.

Reply:
Thanks Martin, let me resolve that